cartographer_ros/CMakeLists.txt

```diff
@@ -88,6 +90,10 @@ set(ALL_SRCS
   "cartographer_ros/trajectory_options.cc"
   "cartographer_ros/submap.cc"
   "cartographer_ros/ros_map.cc"
+  "cartographer_ros/assets_writer.cc"
+  "cartographer_ros/split_string.cc"
+  "cartographer_ros/ros_map_writing_points_processor.cc"
+  "cartographer_ros/urdf_reader.cc"
```

Comment:
Let's rearrange this to be alphabetical as well (it is pretty close already).

Reply:
Done